Tests/004TestCertification.cs

```diff
@@ -0,0 +1,373 @@
+﻿using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Week14CompetitionTaskProjectMars.Models;
+using Week14CompetitionTaskProjectMars.Utilities;
+using AventStack.ExtentReports;
+using AventStack.ExtentReports.Reporter;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+
+
+namespace Week14CompetitionTaskProjectMars.Tests
+{
+  [TestFixture]
+  public class TestCertification : Hooks
+  {
+        private List<LoginModel> _LoginModel;
+        private List<EducationModel> _EducationModel;
+        private List<CertificationModel> _CertificationModel;
+
+     [SetUp]
+     public new void Initialize()
+     {
+       //driver1 = CommonDriver.CreateNewDriver();
+     //  base.Initialize();
```

Comment:
please
remove all commented out code line, its bad practice to have them in code! normal sentences are fine

Reply:
Done